### ISSUE TITLE
Old videos reappear on subscriptions feed despite being marked as watched

### ISSUE THREAD

**Issue by Bukkithead** [YePpHa/YouTubeCenter#1568]
Recently, videos that I know I have watched a few days ago start appearing again on my subscriptions feed. The strange thing is, they are still marked as 'watched' by YouTube center, yet are not being hidden like most of the other videos. This is shown clearly in the attached screenshot, with many videos from 3+ days ago reappearing.

![screenshot 03_17_2015-14_22_17](https://cloud.githubusercontent.com/assets/7946901/6689317/49048884-ccb1-11e4-8fe0-72d1c71d8a0c.jpg)

**Comment by Bukkithead** [comment]
Additionally, I am well aware that I am using a custom userstyle for YouTube, but I would like to clarify that disabling this makes absolutely zero difference to this issue.

**Comment by Yonezpt** [comment]
Try the latest developer version if you aren't using it already, you can download it from here: https://github.com/YePpHa/YouTubeCenter/wiki/Developer-Version

**Comment by Bukkithead** [comment]
Thanks, but that makes no difference at all.

**Comment by Yonezpt** [comment]
The subscription grid remains the same with the latest dev version or are you refering to the watched videos problem?

**Comment by Bukkithead** [comment]
I'm not sure I understand what you mean, but updating to the latest version didn't seem to make any noticeable difference to anything, the feed or the issue I have highlighted.

**Comment by Tester798** [comment]
I have the same problem. Watched videos become unwatched for me.
Numbers on the left under "SUBSCRIPTIONS" which show unwatched videos don't disappear after I navigate to Videos on the channel for some channels (as I was before, started few days ago).
Watched videos got resetted for me regularly.
Not sure if it's related to YTC or if it's YouTube related.

**Comment by Yonezpt** [comment]
@Tester798 In that case disable YTC and check wether the issue still persists. If it does then it is not related to YTC.

**Comment by Tester798** [comment]
@Yonezpt Thanks for your answer! Seems like the issue with counters under "SUBSCRIPTIONS" is indeed caused by YouTube.
But the issue with regular reset watched videos is caused by YTC I believe (because it's YTC's own feature). Few days I got videos grayed out properly with "WATCHED" text on them. But then after few days they got reset. Maybe it's caused by extension autoupdating process? But seems like most settings are preserved, so I'm not sure about this.

**Comment by Yonezpt** [comment]
@Tester798 The watched videos getting unmarked has been often related to YouTube, but YTC does have a feature to remember independently so it shouldn't be affecting if I remember it correctly.

**Comment by Tester798** [comment]
@Yonezpt Yes, I have that setting checked in YTC options, but after some time watched videos got reset for me, don't know why.
Will try to look at the debug log next time I notice it.

**Comment by Tester798** [comment]
Got this again: in log I see video id "3stkrKtf2Ms" in "watchedVideos" and "watchedVideosIndicator" is true, but no badge on the video thumbnail.
"bug" label is for confirmed bugs, right? Tell me if you need more information from my side.

**Comment by Yonezpt** [comment]
@Tester798 Your issue is not the same as the one reported in this thread. This thread is related to watched videos not hiding while yours is related to watched videos being unmarked as watched. It would be best to post on a new thread specific to that issue or in an already existing thread related to the same.

**Comment by leonklingele** [comment]
@YePpHa Why not remove watched videos like this:

    var videos = document.querySelectorAll(".feed-item-container");
    [].forEach.call(videos, function(video) {
    	var watched = video.querySelector(".watched");
    	if (watched) {
    		video.parentNode.remove(video);
    	}
    });

Works perfectly for me.

**Comment by Niddi** [comment]
My watched videos also don't hide anymore. This started at the same time as #1670, so about 3 weeks ago. I've tried reinstalling the userscript, deleting all my Google and Youtube cookies, cleaning the video history and using the addon-version of YTC, but nothing works. The videos are still being marked as watched but they don't auto-hide anymore.

**Comment by Niddi** [comment]
I was able to solve the problem for me. I had to check "UI > Video Thumbnail > Enable Rating Count". If I disable it, my watched videos will not autohide anymore.

**Comment by Bukkithead** [comment]
That seems to work for me too, thank you!

**Comment by Niddi** [comment]
Update: Autohide only works partially. It seems that YTC can only hide ~ 11 - 16 videos. As new videos keep coming in and are being hidden, older videos appear once again (just as the OP described).
